### PR TITLE
fix(@clayui.com): code collapse

### PR DESCRIPTION
**Pull request by bryceosterhaus**
Small updates for https://github.com/liferay/clay/pull/3523

Going to merge and if we need any other updates we can add as necessary, this won't be live un clayui.com till next release

### CODE REVIEW

File: clayui.com/src/components/Editor/index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import React, {useContext, useState} from 'react';
 import {LiveEditor, LiveError, LivePreview, LiveProvider} from 'react-live';
 
 import theme from '../../utils/react-live-theme';
-import useStickyState from '../Hooks/useStickyState';
+import useStateWithLocalStorage from '../Hooks/useStateWithLocalStorage';
```
Comment (bryceosterhaus) [inline clayui.com/src/components/Editor/index.js:15]: @kresimir-coko I renamed this just so we can be explicit as possible

File: clayui.com/src/components/Editor/index.js [hunk 2]
```diff
@@ -104,8 +104,8 @@ const Editor = ({
 							displayType="unstyled"
 							small
 							spritemap={spritemap}
-							symbol={'paste'}
-							title={'Copy'}
+							symbol="paste"
```
Comment (bryceosterhaus) [inline clayui.com/src/components/Editor/index.js:107]: SF

File: clayui.com/src/components/Editor/index.js [hunk 3]
```diff
@@ -131,15 +131,17 @@ const Editor = ({
...
+										}`,
 									}}
 									key={snippet.name}
 									onClick={() => {
-										if (activeIndex !== index) {
-											setCollapseCode(false);
-										} else {
-											setCollapseCode(!collapseCode);
-										}
+										setCollapseCode(
```
Comment (bryceosterhaus) [inline clayui.com/src/components/Editor/index.js:140]: cleaned up the logic a bit to use a ternary

File: clayui.com/gatsby-browser.js [hunk 4]
```diff
@@ -7,12 +7,18 @@
...
+		let showAtlas = true;
+		let showSiteCss = true;
+
+		try {
+			showAtlas = isNullOrTrue(localStorage.getItem('clay.showAtlas'));
+
+			showSiteCss = isNullOrTrue(
+				localStorage.getItem('clay.showSiteCss')
+			);
+		} catch {}
```
Comment (bryceosterhaus) [inline clayui.com/gatsby-browser.js:21]: @kresimir-coko I think this should fix the localstorage issue when it is disabled

**Comment by kresimir-coko** [comment]
Looks good @bryceosterhaus thanks for wrapping it up 👍

File: clayui.com/src/components/Editor/index.js (hunk 1 above)
Comment (wincent) [reply clayui.com/src/components/Editor/index.js:15]: `<devils-advocate>`At the same time, you've leaked the implementation details into the name, arguably making it a leakier abstraction now, and the new name is less descriptive of the purpose and behavior of the hook (ie. before, the word "sticky" conveyed that the state somehow sticks around beyond the normal lifecycle of the component; after, the term "with local storage" conveys that local storage is used but it doesn't really say how/why).`</devils-advocate>`

I have no problem with the change, but I just wanted to point out that "explicit" is not always necessarily better than "abstract".

Comment (wincent) [reply clayui.com/src/components/Editor/index.js:15]: An example I should have mentioned was, what if, for some reason, we decided to switch to `sessionStorage` from `localStorage` in the future? Would you rename the hook to `useStateWithSessionStorage`?

**Pull request merged**